### ISSUE TITLE
Is there a way to tell if the carrier thermostat is online?

### ISSUE THREAD

**Issue by brettonw** [nebulous/infinitude#170]
TL/DR: I want to make a red/green status indicator for whether infinitude is actually communicating with the thermostat

Story:
I have an internal monitor page where I check that various parts of my home are online and I put a red or green indicator for it. I have one for the infinitude server. 

<img width="620" alt="Screenshot 2023-04-03 at 9 41 45 AM" src="https://user-images.githubusercontent.com/2028973/229527699-f8c49afb-3d22-41e7-9961-e638c1e789f2.png">

Recently, my Unifi network has been letting my Infinity thermostats drop off the WiFi network. When it happens, the infinitude server looks like it is still connected and everything appears to be working correctly, so I don't really notice until the temperature changes dramatically or the energy usage spikes (often around 3am). The only way I can get it back online is to reboot the AP (I have a support ticket open with Unifi for this because it just started happening).

What I'd like is to add a condition to the dashboard showing the thermostat is online, ideally if the infinitude server is online with it. 

I noticed a timestamp in notifications.json, but this doesn't necessarily seem to be linked directly to "online" status.

Is there an easy way to do this?

**Comment by nebulous** [comment]
In my case I had to create a dedicated 802.11g network separate from my Unifi devices for the stat to connect/remain connected, so you're not alone with that problem. 

Regarding connectivity...

Each http request/response is stored by Infinitude in the state folder, and each http response will contain a date string, so that's another means to verify connectivity. 

the status endpoint is the most often called, so look in your state folder for the status request/response files
`ls -l state/re*-status*`

`stat state/req-status*` should return the dates for the last time the thermostat contacted Infinitude
`stat state/res-status*` should return the dates for the last time Carrier replied to Infinitude
`grep -a Date: state/res-status*` should also verify the response time from Carrier to Infinitude

so a naïve cli check for connectivity might be
```bash
find state/ -mmin -16|grep req-systems && echo Infinitude alive || echo No request from Infinitude in the last 16 minutes
```

**Comment by brettonw** [comment]
thank you, slight tweak (why 16?):

    find . -mmin -15 | grep req-systems > /dev/null && echo Infinitude alive || echo No request from Infinitude in the last 15 minutes
